utils/db.util.js

```diff
@@ -17,3 +18,20 @@ exports.checkArticleExists = (article_id) => {
       }
     });
 };
+
+exports.commentFormatForInsert = (article_id, username, body) => {
```

Comment:
Apologies if I missed where you used this, but it doesn't look like you need it.

Reply:
You're right, wasn't used. Was just testing an idea out if I were to do db.query insert via a util function. Left in the repo in case I need to use this or something like it later.